pekko-stream/README.md

```diff
@@ -0,0 +1,60 @@
+# Pekko Streams for ReactiveMongo
+
+This is an [Pekko Streams](http://pekko.apache.org) extension for the ReactiveMongo cursors.
+
+## Usage
+
+In your `project/Build.scala`:
+
+```ocaml
+resolvers += "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots/"
+
+libraryDependencies ++= Seq(
+  "org.reactivemongo" %% "reactivemongo" % VERSION,
```

Comment:
!!

Reply:
this is a direct copy of https://github.com/ReactiveMongo/ReactiveMongo-Streaming/blob/master/akka-stream/README.md which also says 'VERSION'

Reply:
I suppose this would need to be `"org.reactivemongo" %% "reactivemongo-pekko" % VERSION,`
if https://github.com/ReactiveMongo/ReactiveMongo/pull/1245 is merged first?

Reply:
at least both alternatives should be mentioned

Reply:
Need update:
- https://oss.sonatype.org/content/repositories/snapshots/org/reactivemongo/reactivemongo-actors-akka_2.11/1.1.0-RC12-SNAPSHOT/reactivemongo-actors-akka_2.11-1.1.0-RC12-SNAPSHOT.pom
- https://oss.sonatype.org/content/repositories/snapshots/org/reactivemongo/reactivemongo-actors-pekko_2.12/1.1.0-RC12-SNAPSHOT/reactivemongo-actors-pekko_2.12-1.1.0-RC12-SNAPSHOT.pom